lib/constants.go

```diff
@@ -968,6 +972,9 @@ var DeSoMainnetParams = DeSoParams{
 	// reserve ratios.
 	CreatorCoinAutoSellThresholdNanos: uint64(10),
 
+	// Unstaked stake can be unlocked after a minimum of N elapsed epochs.
+	StakeLockupEpochDuration: uint64(3),
```

Comment:
will we want to move this to be a param updater thing eventually or nah?

Reply:
Yes, we will. I added a comment. Good call-out.